docs/api.md

```diff
@@ -52,6 +52,32 @@ coupling that inhibits encapsulation & reuse, often making testing difficult.
 
 Override to determine whether the ViewComponent should render.
 
+### #render_in(view_context, &block)
+
+Entrypoint for rendering components.
+
+view_context: ActionView context from calling view
+block: optional block to be captured within the view context
+
+returns HTML that has been escaped by the respective template handler
+
+Example subclass:
+
+app/components/my_component.rb:
+class MyComponent < ViewComponent::Base
+  def initialize(title:)
+    @title = title
+  end
+end
+
+app/components/my_component.html.erb
+<span title="<%= @title %>">Hello, <%= content %>!</span>
+
+In use:
+<%= render MyComponent.new(title: "greeting") do %>world<% end %>
+returns:
+<span title="greeting">Hello, world!</span>
```

Comment:
I'm not sure we should keep the whole comment. Maybe we can remove everything after `respective template handler`?

Reply:
I agree!